src/components/Category.js

```diff
@@ -74,9 +74,11 @@ const CategoryComponent = ({ category, stateContext }) => {
         })
 
         let updatedColumns = columns
+        .map(i => i.key === "Link" ? {...i, render: (link) => <a href={link} target="_blank" rel="noreferrer">{link}</a>} : i )
```

Comment:
Would suggest adding this here - https://github.com/COVID19-Resources-India/covid19resourcesindia/blob/6805d279567509a3fd0cbaf180575c5d2b5d4c00/src/constant/columns.js#L5

Reply:
Thought of that, but didn’t add it there as there was no UI logic in that file.

Reply:
Yes better to add there as we are already looping over there